appConfig.js

```diff
@@ -6,7 +6,7 @@ export default {
   webpackDevServerPort: 3000,
   baseUrl: '',
   ereader: {
-    development: 'http://researchnow-webpub-dev.us-east-1.elasticbeanstalk.com:4444',
+    development: 'http://localhost:4444',
```

Comment:
Will revert this before merge, though, thinking about setting up a flag or something to ease development.  Might do it in the next viewer ticket?

Reply:
A flag to be able to set this for local development (or to provide zero-downtime transitions between readers) seems like a really good idea